java-squid/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -44,4 +46,8 @@ public void checkEndOfExecution(CheckerContext context) {
     // By default do nothing
   }
 
+  public void checkEndOfExecutionPath(CheckerContext context, ConstraintManager constraintManager) {
+    // TODO Auto-generated method stub
```

Comment:
This todo generated by eclipse should not be commited and comment should be replaced.